routes/api.php

```diff
@@ -1,14 +1,14 @@
 <?php
 
 use Illuminate\Support\Facades\Route;
-use App\Http\Controllers\CreateCityController;
-use App\Http\Controllers\DestroyCityController;
-use App\Http\Controllers\GetAllCitiesController;
-use App\Http\Controllers\UpdateCityController;
+use App\Http\Controllers\Cities\StoreCityController;
+use App\Http\Controllers\Cities\DestroyCityController;
+use App\Http\Controllers\Cities\GetAllCitiesController;
+use App\Http\Controllers\Cities\UpdateCityController;
 
 Route::prefix('cities')->name('cities.')->group(function () {
     Route::get('/', GetAllCitiesController::class)->name('index');
-    Route::post('/', CreateCityController::class)->name('store');
-    Route::put('/{id}', UpdateCityController::class)->name('update');
+    Route::post('/', StoreCityController::class)->name('store');
+    Route::patch('/{id}', UpdateCityController::class)->name('update');
```

Comment:
```suggestion
    Route::put('/{id}', UpdateCityController::class)->name('update');
```
`put` was right. The difference between `put` and `patch` is that for `put`, you send all the resource's fields in the request, while with `patch`, you only send specific ones. Here at Light-it, we define update endpoints as `put`.